pkg/alerts/policies_integration_test.go

```diff
@@ -7,13 +7,14 @@ import (
 	"testing"
 
 	nr "github.com/newrelic/newrelic-client-go/internal/testing"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestIntegrationPolicy(t *testing.T) {
+func TestAlertsPolicy_Legacy(t *testing.T) {
 	t.Parallel()
 
-	client := newIntegrationTestClient(t)
+	a := newIntegrationTestClient(t)
```

Comment:
`a` isn't a very descriptive name compared to `client`

Reply:
`client` isn't the client though, it's the Alerts object that has a `client` attached.